usrsctplib/netinet/sctputil.c

```diff
@@ -4693,6 +4698,8 @@ sctp_abort_an_association(struct sctp_inpcb *inp, struct sctp_tcb *stcb,
                           struct mbuf *op_err,
                           int so_locked)
 {
+	uint16_t cause_code;
+	struct sctp_gen_error_cause* cause;
```

Comment:
Please use:
```
struct sctp_gen_error_cause* cause;
#if defined(__APPLE__) && !defined(__Userspace__)
 	struct socket *so;
#endif
uint16_t cause_code;

Reply:
I moved the `struct sctp_gen_error_cause* cause;` below the `#if` as well.